readme.md

```diff
@@ -4,21 +4,23 @@
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
 requests and responses with filters. You can use it out of the box and add your own custom filters and predicates.
-Here's what it does:
 
+###What Skipper Does
 - identifies routes based on the requests' properties, such as path, method, host and headers
 - routes each request to the configured server endpoint
 - allows alteration of requests and responds with filters that are independently configured for each route
 - optionally acts as a final endpoint (shunt)
-- updates the routing rules without restarting, while supporting multiple types of data sources — including
-  [Innkeeper](https://github.com/zalando/innkeeper), [etcd](https://github.com/coreos/etcd) and static files
+- updates the routing rules without restarting, while supporting multiple types of data sources — including [etcd](https://github.com/coreos/etcd), [Innkeeper](https://github.com/zalando/innkeeper) and static files
 
-Skipper's design is largely inspired by [Vulcand](https://github.com/vulcand/vulcand).
+####Inspiration
+Skipper's design is largely inspired by [Vulcand](https://github.com/vulcand/vulcand). 
 
+### Getting Started
+####Prerequisites/Requirements
+- [What technologies/special versions?]
```

Comment:
build dependency: latest version of go, currently 1.6
optional runtime environment dependency: innkeeper, etcd